### PR TITLE
Append charge chgcar function

### PR DESCRIPTION
**Pull request by jan-janssen**

### CODE REVIEW

**Comment by FeLoch** [comment]
If we append those new functions, is there any need for the already existing `copy_chgcar` and `copy_wavecar`?

**Comment by jan-janssen** [comment]
You are right, those were included for backwards compatibility, but I guess we can remove them now.

File: pyiron/vasp/base.py [hunk 1]
```diff
@@ -1065,15 +1065,37 @@ def restart_from_charge_density(self, snapshot=-1, job_name=None, job_type=None,
...
+            return new_ham
+        else:
+            return new_ham
+
+    def append_charge_density(self, job_specifier=None, path=None):
+        """
+        Append charge density file (CHGCAR)
+
+        Args:
+            job_specifier (str, int): name of the job or job ID
```
Comment (sudarsan-surendralal) [inline pyiron/vasp/base.py]: `job_specifier (str/int)` is the right way

Comment (FeLoch) [reply pyiron/vasp/base.py]: done

**Pull request merged**